modules/ROOT/pages/7.3-release-notes.adoc

```diff
@@ -61,10 +61,14 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 {productname} {release-version} also includes the following improvement<s>:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== When a full document was loaded as editor content the head elements were added to the body.
+// #TINY-11053
 
-// CCFR here.
+Previously when loading a full document as editor content, head elements were incorrectly added to the body. This issue occurred because the browser detected HTML, head, and body tags within the wrapped body block, causing them to be ignored and their content placed incorrectly.
```

Comment:
A bit clunky, but also not quite sure how to explain it well. At the root it was because we wrapped the html in a `body` tag ( for various reasons ) before we parsed it. As `html`, `head`, and `body` tags are not supposed to be inside of a `body` tag at all they were unwrapped, might be the best way of calling it. The content of these elements moved to the wrapping `body` tag we'd added. So `<html><head><HeadContent /></head><body><BodyContent /></body>` was treated as if it had been just `<HeadContent /><BodyContent />` during the parsing of the text. Which caused the issue as we can't ignore the content of the `head` if there's no way to tell where it came from.
The solution, for the record, was to try and detect these situations and wrap them in a `html` instead of a `body`. It still unwraps the `html` provided if any, but that shouldn't pose any problems.
As for how to put any of this in the documentation, I don't know.